README.rst

```diff
@@ -670,3 +670,13 @@ To include a ``"foo.cfg"`` config file, use the following statement::
     include("foo.cfg")
 
 This allows to re-use, for example flash configuration files, across different image configurations.
+
+# To develop and run the test
+
+```bash
+./autogen.sh
+./configure CFLAGS='-g -O0' --prefix=/usr
```

Comment:
This whole change should be a separate commit.

And don't suggest '-O0' here: Quite a few compiler warnings are skipped in that case. Just remove the CFLAGS, the default '-g -O2' should work just fine.

Reply:
Ok im go to change that

Reply:
Fixed